pkg/cloudprovider/providers/azure/azure_routes.go

```diff
@@ -97,6 +97,12 @@ func (az *Cloud) createRouteTableIfNotExists(clusterName string, kubeRoute *clou
 }
 
 func (az *Cloud) createRouteTable() error {
+	var rg string
+	if len(az.RouteTableResourceGroup) > 0 {
```

Comment:
Can you add a test case to cover defaulting the RG

Reply:
I'm pretty a newbee. Where can I find an example ? Thanks !

Reply:
Done !